lib/handleRuntimeProblems.js

```diff
@@ -27,10 +30,11 @@ module.exports = function handleRuntimeProblems(blueprintData) {
         log(message);
       } else {
         const transactionName = [
+          annotation.origin.apiName,
```

Comment:
We're adding new component to the transaction name which is a feature? This commit is tagged as a fix.

Reply:
It is only for the purpose of printing it out in the annotation. So if the annotation text is considered to be a public API, then this should be `feat`, but otherwise no. I don't know why this code creates the transaction name from scratch, but since I'm about to delete it, I don't care that much.